src/assets/styles/20-tools/_extends/_colours.scss

```diff
@@ -0,0 +1,94 @@
+// ----------------------------------
+// UI Tools
+// Extends
+// Colours
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+// Background Colours
+
+%bg-colour-black {
```

Comment:
Not sure how useful these extends are. If there were more style rules to be applied with a more abstracted name it would be worth it certainly

Reply:
Yeah that's fair enough, they would be useful as utility classes I think but I also think it's best not to have too many utility classes in pattern library.

Happy enough parking this for now 👍

Reply:
Actually now I come to think of it - this would save us lines of code in the built CSS file, so could be worth doing after all from a performance POV?